tests/test_cli.py

```diff
@@ -85,3 +92,118 @@ def test_api_method_not_found(nyse_tickers, capfd):
+    testing.
+    """
+    with tempfile.TemporaryDirectory() as tempdir:
+        yield os.path.join(tempdir, 'piker')
+
+
+@pytest.fixture
+def piker_dir(temp_dir):
+    wl.make_config_dir(temp_dir)
+    json_file_path = os.path.join(temp_dir, 'watchlists.json')
+    watchlist = {
+        'dad': ['GM', 'TSLA', 'DOL.TO', 'CIM', 'SPY', 'SHOP.TO'],
+        'pharma': ['ATE.VN'],
+        'indexes': ['SPY', 'DAX', 'QQQ', 'DIA'],
+    }
+    wl.write_sorted_json(watchlist, json_file_path)
+    yield json_file_path
+
+
+def test_show_watchlists(capfd, piker_dir):
+    """Ensure a watchlist is printed.
+    """
+    expected_out = json.dumps({
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'indexes': ['DAX', 'DIA', 'QQQ', 'SPY'],
+        'pharma': ['ATE.VN'],
+    }, indent=4)
+    run(f"piker watchlists -d {piker_dir} show")
+    out, err = capfd.readouterr()
+    assert out.strip() == expected_out
+
+
+def test_dump_watchlists(capfd, piker_dir):
+    """Ensure watchlist is dumped.
+    """
+    expected_out = json.dumps({
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'indexes': ['DAX', 'DIA', 'QQQ', 'SPY'],
+        'pharma': ['ATE.VN'],
+    })
+    run(f"piker watchlists -d {piker_dir} dump")
+    out, err = capfd.readouterr()
+    assert out.strip() == expected_out
+
+
+def test_ticker_added_to_watchlists(capfd, piker_dir):
+    expected_out = {
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'indexes': ['DAX', 'DIA', 'QQQ', 'SPY'],
+        'pharma': ['ATE.VN', 'CRACK'],
+    }
+    run(f"piker watchlists -d {piker_dir} add pharma CRACK")
+    out = wl.ensure_watchlists(piker_dir)
+    assert out == expected_out
+
+
+def test_ticker_removed_from_watchlists(capfd, piker_dir):
+    expected_out = {
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'indexes': ['DAX', 'DIA', 'SPY'],
+        'pharma': ['ATE.VN'],
+    }
+    run(f"piker watchlists -d {piker_dir} remove indexes QQQ")
+    out = wl.ensure_watchlists(piker_dir)
+    assert out == expected_out
+
+
+def test_group_deleted_from_watchlists(capfd, piker_dir):
+    expected_out = {
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'indexes': ['DAX', 'DIA', 'QQQ', 'SPY'],
+    }
+    run(f"piker watchlists -d {piker_dir} delete pharma")
+    out = wl.ensure_watchlists(piker_dir)
+    assert out == expected_out
+
+
+def test_watchlists_loaded(capfd, piker_dir):
+    expected_out_text = json.dumps({
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'pharma': ['ATE.VN'],
+    })
+    expected_out = {
+        'dad': ['CIM', 'DOL.TO', 'GM', 'SHOP.TO', 'SPY', 'TSLA'],
+        'pharma': ['ATE.VN'],
+    }
+    run(f"piker watchlists -d {piker_dir} load", expected_out_text)
+    out = wl.ensure_watchlists(piker_dir)
+    assert out == expected_out
+
+
+def test_watchlists_is_merge(capfd, piker_dir):
```

Comment:
In this test you should verify that if two watchlists with lists of the same name are merge then the elements (tickers) from each are merged as well.